packages/@ama-sdk/schematics/schematics/ng-add/index.ts

```diff
@@ -74,7 +74,7 @@ export const updatePackageJsonScripts: Rule = (tree, context) => {
             /@ama-sdk\/(schematics|generator-sdk):(core|shell|create|mock)/g,
             '@ama-sdk/schematics:typescript-$2'
           ) // Change typescript schematics name
-          .replaceAll('--swaggerSpecPath', '--swagger-spec-path') // Schematics arguments should be kebab-case
+          .replaceAll('--specPath', '--spec-path') // Schematics arguments should be kebab-case
```

Comment:
```suggestion
          .replaceAll('--swaggerSpecPath', '--spec-path') // Schematics arguments should be kebab-case
```

Reply:
why when we support both swagger and open api spec?